src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -102,48 +101,55 @@ def generate_MCP_1_default_config(self, filename):
     pal_low_frequency = 3550000000.0
     pal_high_frequency = 3560000000.0        
 
-    ppa_record_1, pal_record_1 = makePpaAndPalRecordsConsistent(ppa_record,
+    ppa_records_1, pal_records_1 = makePpaAndPalRecordsConsistent(ppa_record,
```

Comment:
There is only one PPA record returned from this function.

Reply:
Modified   to ppa_record_1